openassessment/assessment/api/staff.py

```diff
@@ -3,11 +3,14 @@
 """
 import logging
 from django.db import DatabaseError, IntegrityError, transaction
+from django.utils.timezone import now
 from dogapi import dog_stats_api
 
+from submissions import api as submissions_api
+
```

Comment:
Well, since you asked...

1. I follow the PEP8 guidance to group logical sets of imports (see https://github.com/edx/edx-platform/wiki/Python-Guidelines)
2. I put a line between each logical group
3. Within the group, I sort the imports alphabetically

Your mileage may vary as to whether this is too anal :-)